python/pyspark/sql/streaming/query.py

```diff
@@ -387,7 +390,7 @@ def exception(self) -> Optional[StreamingQueryException]:
             je = self._jsq.exception().get()
             msg = je.toString().split(": ", 1)[1]  # Drop the Java StreamingQueryException type info
             stackTrace = "\n\t at ".join(map(lambda x: x.toString(), je.getStackTrace()))
-            return StreamingQueryException(msg, stackTrace, je.getCause())
+            return CapturedStreamingQueryException(msg, stackTrace, je.getCause())
```

Comment:
Just for clear understanding, will it raise `StreamingQueryException` in user space, right ?
I just want to clarify because we don't use such an alias for connect exceptions for example:
```python
from pyspark.errors.exceptions.connect import (
    AnalysisException as ConnectAnalysisException
)
```

Reply:
The name `StreamingQueryException` is already imported for the return type annotation, so we just need to use another name.

https://github.com/apache/spark/blob/673d2de773a3c05e3e1325e727d31a7aa403186d/python/pyspark/sql/streaming/query.py#L24-L27